app/src/test/kotlin/compiler/semantic_analysis/VariablePropertiesAnalyzerTest.kt

```diff
@@ -72,7 +72,7 @@ class VariablePropertiesAnalyzerTest {
         val input = VariablePropertyInput(Program(listOf(VariableDefinition(variable))), referenceMapOf())
 
         val expectedResults: ReferenceMap<Any, VariableProperties> = referenceMapOf(
-            variable to VariableProperties(null, referenceSetOf(), referenceSetOf()),
+            variable to VariableProperties(VariablePropertiesAnalyzer.GlobalContext, referenceSetOf(), referenceSetOf()),
```

Comment:
This test should be renamed as now that variable has a parent (assuming parent = owner here).